lib/view_model/after_auth_view_models/profile_view_models/profile_page_view_model.dart

```diff
@@ -84,9 +85,16 @@ class ProfilePageViewModel extends BaseModel {
                 url.clear();
                 androidFirebaseOptionsBox.clear();
                 iosFirebaseOptionsBox.clear();
-                Firebase.app()
-                    .delete(); // Deleting app will stop all Firebase plugins
+                try {
+                  Firebase.app()
+                      .delete(); // Deleting app will stop all Firebase plugins
+                } catch (e) {
+                  debugPrint(
+                      "ERROR: Unable to delete firebase app" + e.toString());
+                }
+
```

Comment:
Fix: Firebase throws error if it's not initialized previously. Added a `try {..} catch{..} ` to fix it,